Cargo.toml

```diff
@@ -1,5 +1,5 @@
 [package]
-name = "Cosmo"
+name = "cosmo"
 version = "0.1.0"
 edition = "2021"
 
```

Comment:
Ajouter le nom de l'auteur : https://doc.rust-lang.org/cargo/reference/manifest.html#the-authors-field